lib/mapping/CMapInfo.cpp

```diff
@@ -45,7 +45,7 @@ CMapInfo::CMapInfo() : scenarioOpts(nullptr), playerAmnt(0), humanPlayers(0),
 
 #define STEAL(x) x = std::move(tmp.x)
 
-CMapInfo::CMapInfo(CMapInfo && tmp):
+CMapInfo::CMapInfo(CMapInfo & tmp):
 	scenarioOpts(nullptr), playerAmnt(0), humanPlayers(0),

```

Comment:
Why you broken move constructor?

Reply:
It's was conflicting with some of my shared_ptr code during refactoring and the only thing commit description on it mentioned is that it was implemented to workaround some VS2013 bug.

Is it really needed?

Reply:
`+CMapInfo::CMapInfo(CMapInfo & tmp)` is a copy constructor now, but it still works like move constructor.